### PR TITLE
Update addCourseMgmt.sql

### PR DESCRIPTION
**Pull request by griffine**
Added 'modifyCourse' function and 'courseExists' function to addCourseMgmt.sql.

Implemented the 'courseExists' function into the 'insertCourse' and 'removeCourse' functions in place of the previous 'IF EXISTS (...)' tests

Changed around some of the formatting of lines to be more readable and clean.

### CODE REVIEW

File: src/db/addCourseMgmt.sql [hunk 1]
```diff
@@ -38,31 +32,109 @@ LANGUAGE plpgsql;
...
+RETURNS BOOL AS
+$$
+BEGIN
 
+   -- evaluate if no credits are given
+   IF cCredits IS NULL
+   THEN 
+      IF EXISTS 
+      (
+         SELECT * FROM Course WHERE Course.Number = cNumber
```
Comment (kozlowskik) [inline src/db/addCourseMgmt.sql]: This will probably need to be changed to say `Gradebook.Course`

File: src/db/addCourseMgmt.sql [hunk 2]
```diff
@@ -38,31 +32,109 @@ LANGUAGE plpgsql;
...
+      )
+      THEN
+         RETURN true;
+      END IF;
+   END IF;
+   
+   -- evaluate if credits are given
+   IF EXISTS
+   (
+      SELECT * FROM Course WHERE Course.Number = cNumber
```
Comment (kozlowskik) [inline src/db/addCourseMgmt.sql]: This too will probably need to say `Gradebook.Course`

**Review by kozlowskik — CHANGES_REQUESTED** [review]

**Review by kozlowskik — APPROVED** [review]

**Review by DasilvaB — APPROVED** [review]
looks good

**Review by fitz-cris — APPROVED** [review]
good catch Ken, otherwise looks good to go

**Pull request merged**